docs.html

```diff
@@ -20,104 +21,195 @@
       </a>
     </div>
     <div class="nav-header-right-side">
       <a href="/">
         <div class="nav-header-text">Documentation</div>
       </a>
-      <div class="toggler "><i class="fas fa-moon"></i></div>
+      <div class="toggler header-icon"><i class="fas fa-moon"></i></div>
     </div>
-
   </div>
 
-
   <div class="nav-header nav-header-desktop hide">
     <a href="/">
-      <div class="logo">IndoUI</div>
+      <div class="logo">PabbleUI</div>
     </a>
     <div class="nav-header-right-side">
       <a href="/">
         <div class="nav-header-text">Documentation</div>
       </a>
-      <div class="toggler "><i class="fas fa-moon"></i></div>
+      <a href="https://github.com/saurabhchirde/Pabble-UI" target="_blank">
+        <div class="header-icon"><i class="fab fa-github"></i></div>
+      </a>
+      <a href="https://twitter.com/SaurabhChirde" target="_blank">
+        <div class="header-icon"><i class="fab fa-twitter"></i></div>
+      </a>
+      <div class="toggler header-icon"><i class="fas fa-moon"></i></div>
     </div>
   </div>
 
-  <!-- nav verticle bar -->
-  <nav class="nav">
+  <!-- nav verticle bar desktop -->
+  <nav class="nav-desktop">
     <a href="#">
-      <h1 class="nav-category pd-point8-lt">Getting Started</h1>
+      <h1>Getting Started</h1>
     </a>
     <hr class="nav-break-line">
     <a href="./Documentation/docpage/about.html">
-      <h1 class="nav-category pd-point8-lt">About</h1>
+      <h1>About</h1>
     </a>
     <a href="./Documentation/docpage/color.html">
-      <h1 class="nav-category pd-point8-lt">Color</h1>
+      <h1>Color</h1>
     </a>
     <a href="./Documentation/docpage/typography.html">
-      <h1 class="nav-category pd-point8-lt">Typography</h1>
+      <h1>Typography</h1>
     </a>
     <hr class="nav-break-line">
     <a href="./Documentation/docpage/components-overview.html">
-      <h1 class="nav-category pd-point8-lt 
-      nav-category-active">Component Overview</h1>
+      <h1>Component Overview</h1>
     </a>
     <hr class="nav-break-line">
-    <div class="nav-items">
-      <a href="./Documentation/docpage/alert.html">
-        <h2 class="nav-alert nav-btn">Alerts</h2>
-      </a>
-      <a href="./Documentation/docpage/buttons.html">
-        <h2 class="nav-button nav-btn">Buttons</h2>
-      </a>
-      <a href="./Documentation/docpage/avatar.html">
-        <h2 class="nav-avatar nav-btn">Avatar</h2>
-      </a>
-      <a href="./Documentation/docpage/badge.html">
-        <h2 class="nav-badge nav-btn">Badge</h2>
-      </a>
-      <a href="./Documentation/docpage/cards.html">
-        <h2 class="nav-card nav-btn">Cards</h2>
-      </a>
-      <a href="./Documentation/docpage/image.html">
-        <h2 class="nav-img nav-btn">Image</h2>
-      </a>
-      <a href="./Documentation/docpage/input.html">
-        <h2 class="nav-input nav-btn">Input</h2>
-      </a>
+    <a href="./Documentation/docpage/alert.html">
+      <h1 class="nav-alert nav-btn">Alerts</h1>
```

Comment:
For class names of items in the nav list, are different style classes needed? please revisit this, unless you're adding a custom style to each list item, 
"nav-alert, nav-button, nav-badge" etc

Reply:
Thank you for pointing this out, I had already applied styles on them using the child selector. So removed both of them.